### PR TITLE
platform-configs/: added MTL platform configs

### PR DESCRIPTION
**Pull request by JanPrusinowski**
README.md: added platforms to the list
variables.robot: added platform configs

### CODE REVIEW

File: platform-configs/include/novacustom-mtl.robot [hunk 1]
```diff
@@ -0,0 +1,30 @@
+*** Variables ***
+# Flash
+${FLASH_SIZE}=                      ${16*1024*1024}
```
Comment (mkopec) [inline platform-configs/include/novacustom-mtl.robot:3]:
```suggestion
${FLASH_SIZE}=                      ${32*1024*1024}
```

File: platform-configs/include/novacustom-mtl.robot [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+${DEF_SOCKETS}=                     1
+
+# Audio
+${DEVICE_AUDIO1}=                   ALC245
+${DEVICE_AUDIO2}=                   Tigerlake HDMI
+${DEVICE_AUDIO1_WIN}=               Realtek High Definition Audio
+
+# Connectivity
+${WIFI_CARD}=                       Intel(R) Wi-Fi 6 AX201 160MHz
+${WIFI_CARD_UBUNTU}=                Intel Corporation Wi-Fi 6 AX201 (rev 20)
```
Comment (mkopec) [inline platform-configs/include/novacustom-mtl.robot:21]: I think the models we got should have AX211 IIRC

```suggestion
${WIFI_CARD_UBUNTU}=                Intel Corporation Wi-Fi 6 AX211 (rev 20)
```

Should be listed in `lspci`

File: platform-configs/include/novacustom-mtl.robot [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+${DEF_CORES_PER_SOCKET}=            4
+${DEF_THREADS_PER_CORE}=            2
+${DEF_THREADS_TOTAL}=               ${EMPTY}
+# TODO: remove, the value below can be inferred from the one above
+${DEF_ONLINE_CPU}=                  0-7
+${DEF_SOCKETS}=                     1
+
+# Audio
+${DEVICE_AUDIO1}=                   ALC245
+${DEVICE_AUDIO2}=                   Tigerlake HDMI
```
Comment (mkopec) [inline platform-configs/include/novacustom-mtl.robot:16]: Should be something like `Meteor Lake-P HDMI`. You can obtain it with `cat /proc/asound/card0/codec\#2 | grep -i "Codec:"`

File: platform-configs/include/novacustom-mtl.robot [hunk 4]
```diff
@@ -0,0 +1,30 @@
+*** Variables ***
+# Flash
+${FLASH_SIZE}=                      ${16*1024*1024}
+
+# CPU
+${INITIAL_CPU_FREQUENCY}=           2800
+${DEF_CORES_PER_SOCKET}=            4
```
Comment (mkopec) [inline platform-configs/include/novacustom-mtl.robot:7]: The 155H models should be 16C / 22T if I'm not mistaken

File: platform-configs/include/novacustom-mtl.robot [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+${DEF_THREADS_PER_CORE}=            2
```
Comment (mkopec) [inline platform-configs/include/novacustom-mtl.robot:8]: This variable doesn't work with heterogeneous CPUs like meteorlake, but that's a problem for later.

File: platform-configs/include/novacustom-mtl.robot [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+# Audio
+${DEVICE_AUDIO1}=                   ALC245
+${DEVICE_AUDIO2}=                   Tigerlake HDMI
+${DEVICE_AUDIO1_WIN}=               Realtek High Definition Audio
+
+# Connectivity
+${WIFI_CARD}=                       Intel(R) Wi-Fi 6 AX201 160MHz
+${WIFI_CARD_UBUNTU}=                Intel Corporation Wi-Fi 6 AX201 (rev 20)
+${BLUETOOTH_CARD_UBUNTU}=           Intel Corp. AX201 Bluetooth
```
Comment (mkopec) [inline platform-configs/include/novacustom-mtl.robot:22]: a/a, you can get this one from `lsusb`

File: platform-configs/novacustom-v560tu.robot [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+
+${3_MDEB_WIFI_NETWORK}=             3mdeb_abr
+${DEVICE_NVME_DISK}=                Non-Volatile memory controller
+${DEVICE_USB_KEYBOARD}=             Logitech, Inc. Keyboard K120
+${DMIDECODE_PRODUCT_NAME}=          V560TU
+${EXTERNAL_HEADSET}=                USB PnP Audio Device
+${CPU_MAX_FREQUENCY}=               4500
+${CPU_MIN_FREQUENCY}=               300
+
+${NVIDIA_GRAPHICS_CARD_SUPPORT}=    ${TRUE}
```
Comment (mkopec) [inline platform-configs/novacustom-v560tu.robot:18]: Only the `*TNx` models have Nvidia graphics, the `TU` models have integrated graphics only.

**Review by mkopec — CHANGES_REQUESTED** [review]

File: platform-configs/include/novacustom-mtl.robot (hunk 3 above)
Comment (JanPrusinowski) [reply platform-configs/include/novacustom-mtl.robot]: fixed in: ebada758dbf9ba9229c9a44163d599ee96603f52

File: platform-configs/include/novacustom-mtl.robot (hunk 6 above)
Comment (JanPrusinowski) [reply platform-configs/include/novacustom-mtl.robot]: fixed in: ebada758dbf9ba9229c9a44163d599ee96603f52

File: platform-configs/novacustom-v560tu.robot (hunk 7 above)
Comment (JanPrusinowski) [reply platform-configs/novacustom-v560tu.robot]: fixed in: ebada758dbf9ba9229c9a44163d599ee96603f52

**Review by mkopec — APPROVED** [review]

**Pull request merged**